test/pagespecs/pages/NotEnrolledPage.scala

```diff
@@ -70,7 +70,8 @@ class NotEnrolledPage(baseUrl: BaseUrl)(implicit webDriver: WebDriver) extends B
           |Before you call, make sure you have:
           |your Self Assessment Unique Taxpayer Reference (UTR) which is 10 digits long, like 1234567890
```

Comment:
is this right? Looks like the test is not testing the line `your bank details` now and is testing the line `details of your income` appears twice